data/grad_employ_subset.R

```diff
@@ -0,0 +1,13 @@
+delayedAssign("grad_employ_subset", local({
+  if (requireNamespace("epiprocess", quietly = TRUE)) {
+    d <- as.Date("2024-09-18")
+    epiprocess::as_epi_df(
+      epidatasets:::grad_employ_subset_dt,
+      as_of = d,
+      additional_metadata = list(other_keys = c("age_group", "edu_qual"))
```

Comment:
```suggestion
      other_keys = c("age_group", "edu_qual")
```
plus bump epiprocess version in renv + bump / place an epiprocess version range in Enhances: if possible.